packages/suite/src/reducers/wallet/sendFormReducer.ts

```diff
@@ -22,49 +23,50 @@ export const initialState: SendState = {
 
 const sendFormReducer = (state: SendState = initialState, action: Action): SendState =>
     produce(state, draft => {
-        switch (action.type) {
-            case STORAGE.LOAD:
-                action.payload.sendFormDrafts.forEach(d => {
-                    draft.drafts[d.key] = d.value;
-                });
-                break;
-            case SEND.STORE_DRAFT:
-                draft.drafts[action.key] = action.formState;
-                break;
-            case SEND.REMOVE_DRAFT:
-                delete draft.drafts[action.key];
-                break;
-            case ACCOUNT.REMOVE:
-                action.payload.forEach(account => {
-                    delete draft.drafts[account.key];
-                });
-                break;
-            case SEND.REQUEST_SIGN_TRANSACTION:
-                if (action.payload) {
-                    draft.precomposedTx = action.payload.transactionInfo;
-                    draft.precomposedForm = action.payload.formValues;
-                } else {
+        if (accountsActions.removeAccount.match(action)) {
```

Comment:
in case of reducer please use as we did it before, place if into switch, I don't want to mess convention of using only switch in reducer